util/topgen/templates/toplevel_memory.ld.tpl

```diff
@@ -2,6 +2,8 @@
 /* Licensed under the Apache License, Version 2.0, see LICENSE for details. */
 /* SPDX-License-Identifier: Apache-2.0 */
 <%!
+import topgen.lib as lib
```

Comment:
I don't see a use of `lib` in the modified code. Is this needed?

Reply:
Not needed, the usage of `lib` was removed in this commit series. I removed the import